pkg/exchange/okex/stream.go

```diff
@@ -192,6 +185,17 @@ func (s *Stream) handleCandleEvent(candle Candle) {
 
 func (s *Stream) createEndpoint(ctx context.Context) (string, error) {
 	var url string
+	for _, subscription := range s.Subscriptions {
+		sub, err := convertSubscription(subscription)
+		if err != nil {
+			log.WithError(err).Errorf("subscription convert error")
+			continue
+		}
+
+		if strings.HasPrefix(strings.ToLower(string(sub.Channel)), "candle") {
```

Comment:
what's the purpose here?

Reply:
because OKX for candlestick event No longer use public endpoint. Please refer to https://www.okx.com/hk/help/changes-to-v5-api-websocket-subscription-parameter-and-url 
So we need to discuss if we encounter candlestick event, we just pass an error or we handle endpoint for it ?
cc @bailantaotao